kibana/kibana.yml

```diff
@@ -1,67 +1,79 @@
+# If your Elasticsearch is protected with basic auth, these are the user credentials
+# used by the Kibana server to perform maintenance on the kibana_index at startup. Your Kibana
+# users will still need to authenticate with Elasticsearch (which is proxied through
 # the Kibana server)
-kibana_elasticsearch_username: kibana
-kibana_elasticsearch_password: changeme
+#elasticsearch.username: "kibana"
+#elasticsearch.password: "changeme"
 
-# If your Elasticsearch requires client certificate and key
-kibana_elasticsearch_client_crt: /etc/kibana/keys/cert
-kibana_elasticsearch_client_key: /etc/kibana/keys/key
+# SSL for outgoing requests from the Kibana Server to the browser (PEM formatted)
+# server.ssl.cert: /path/to/your/server.crt
+# server.ssl.key: /path/to/your/server.key
 
-# If you need to provide a CA certificate for your Elasticsarech instance, put
+# Optional setting to validate that your Elasticsearch backend uses the same key files (PEM formatted)
+elasticsearch.ssl.cert: /etc/kibana/keys/cert
+elasticsearch.ssl.key: /etc/kibana/keys/key
+
+# If you need to provide a CA certificate for your Elasticsearch instance, put
 # the path of the pem file here.
-ca: /etc/kibana/keys/ca
+elasticsearch.ssl.ca: /etc/kibana/keys/ca
 
-# The default application to load.
-default_app_id: "discover"
+# Set to false to have a complete disregard for the validity of the SSL
+# certificate.
+# elasticsearch.ssl.verify: true
+
+# Time in milliseconds to wait for elasticsearch to respond to pings, defaults to
+# request_timeout setting
+#elasticsearch.pingTimeout: 15000
 
 # Time in milliseconds to wait for responses from the back end or elasticsearch.
 # This must be > 0
-request_timeout: 300000
+elasticsearch.requestTimeout: 30000
 
 # Time in milliseconds for Elasticsearch to wait for responses from shards.
 # Set to 0 to disable.
-shard_timeout: 0
-
-# Set to false to have a complete disregard for the validity of the SSL
-# certificate.
-verify_ssl: true
+# elasticsearch.shardTimeout: 0
 
-# SSL for outgoing requests from the Kibana Server (PEM formatted)
-# ssl_key_file: /path/to/your/server.key
-# ssl_cert_file: /path/to/your/server.crt
+# Time in milliseconds to wait for Elasticsearch at Kibana startup before retrying
+# elasticsearch.startupTimeout: 5000
 
 # Set the path to where you would like the process id file to be created.
-# pid_file: /var/run/kibana.pid
-
-# Plugins that are included in the build, and no longer found in the plugins/ folder
-bundled_plugin_ids:
- - plugins/dashboard/index
- - plugins/discover/index
- - plugins/doc/index
- - plugins/kibana/index
- - plugins/markdown_vis/index
- - plugins/metric_vis/index
- - plugins/origin-kibana/index
- - plugins/settings/index
- - plugins/table_vis/index
- - plugins/vis_types/index
- - plugins/visualize/index
+# pid.file: /var/run/kibana.pid
+
+# If you would like to send the log output to a file you can set the path below.
+# logging.dest: stdout
```

Comment:
the default is stdout which is what Docker is expecting.  Its commented out and kept around just as a 'fyi' that it can be changed (if need be) in the future